Documentation/How to Prepare a Publish Profile.md

```diff
@@ -0,0 +1,64 @@
+# How to Prepare a Publish Profile
+
+In order to deploy applications to a development board (e.g. Raspberry Pi, HummingBoard, etc.), you need to setup a Publish Profile.  Below provides a few steps to quickly get started wihin your projects.
```

Comment:
I wonder if we should make this doc a more general one which will contain instructions for just publishing to a board. That way, we would have two methods described: Publish Profiles when using Visual Studio, and dotnet-publish when using command-line tools and VS Code.

That way, this doc will be helpful as well for people that are just trying to get a sample app to run on their Pi and don't have Visual Studio installed.

Reply:
So are you saying...
1. Rename the doc to something like "How to Publish App to Dev Board"?
2. Do we keep the current contents as the first section and then section two for moving app files to board?
3. Is there a particular process you use on move app to board?  I've been using Filezilla, but not sure if that is the best approach.
4. Not sure tooling around VS Code, either.  Any helpful links available?

Reply:
Yes, renaming the doc to be more of a full guide on how to get an app from the development machine into the actual board would be great. 

Keeping the existing contents as the section of how to publish from VS is fine. Then we can add another section of how to do this without VS. In order to do that, you don't really do it through VS Code, but simply run from the command line: `dotnet build` and then `dotnet publish -o <outputFolder> -r <RuntimeIdentifierYouAreTargeting>` which should do the trick.

I have been mainly developing on a Windows machine, so I usually use putty tools to ssh into the device. When installing putty, another tool called pscp is installed as well which helps you transfer files through ssh. That said, we don't have to mention all the possible ways of doing it, as long as we have one reliable way of doing it on the guide that works with Windows, and one with Linux that should be good enough.